driverthroughput/main.py

```diff
@@ -1,26 +1,30 @@
-import fire
 from utils import *
+from driverprocessresults.main import *
+
+if __name__ == "__main__":
+    sys.path.append("..")
 
 from smartsim.log import get_logger, log_to_file
 logger = get_logger("Scaling Tests")
 
 class Throughput:
     def throughput_standard(self,
-                           exp_name="throughput-standard-scaling",
+                           exp_name="inference-standard-scaling",
```

Comment:
this is throughput, not inference. Why the change?

Reply:
thank you for catching that!